pages/quests/quest1.mdx

```diff
@@ -105,14 +110,18 @@ Handlers.add(
 
 <i>**6.2: Checking Process Owner and Extracting Token Information**</i>
 
+In this snippet, msg.From and msg.Tags are utilized to access a particular attribute within the structured message. For further reference on the specific structure of the message, visit <a href="https://cookbook_ao.arweave.dev/concepts/messages.html" style={{color: "orange", textDecoration: "underline"}}>here</a>.
+
 ```lua
 if msg.From == ao.id then
 	local token = msg.Tags.Token
-  local coingecko_id = msg.Tags.CoingeckoId
+    local coingecko_id = msg.Tags.CoingeckoId
 ```
 
 <i>**6.3: Adding token to TOKEN_PRICES Table**</i>
 
+These lines of code first check if a token exists in the TOKEN_PRICES table. If not, it initializes its price to 0 and sets its coingecko_id. Then, it sends a success message to msg.From with relevant tags. Otherwise, it sends an error message indicating that the token already exists.
```

Comment:
- TOKEN_PRICES, msg.From should be in `code` syntax

Reply:
- ~These lines of code first check if a token exists in the TOKEN_PRICES table~ the following code checks if the token exists in the `TOKEN_PRICES` table